pyppl/runner.py

```diff
@@ -6,8 +6,9 @@
 from os import getcwd
 from multiprocessing import Lock
 from psutil import pid_exists
+import cmdy
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'cmdy'](https://app.codacy.com/app/pwwang/PyPPL/pullRequest?prid=4523419)